### DISCUSSION TITLE
求助，添加onedriveapp存储，提示Unable to retrieve user's mysite URL

### DISCUSSION THREAD

**Discussion by skyflying1989** [AlistGo/alist#4524] [General]
按照文档一步步添加onedriveapp存储，添加成功后，打开却提示Unable to retrieve user's mysite URL，这是怎么回事，该怎么处理

**Comment by anwen-anyi** [comment]
新添加的用户不能立刻生效有延迟 需要等等

**Reply by Echo052118** [reply]
> 新添加的用户不能立刻生效有延迟 需要等等

failed get objs: failed to list objs: User's mysite not found.
那这是咋回事，没添加好？找不到mysite
